README.md

```diff
@@ -262,6 +262,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Drawing
 
+* [**FreePaint**](https://github.com/pastthepixels/FreePaint) <sup>**[[F-Droid](https://www.f-droid.org/app/io.github.pastthepixels.freepaint)]**</sup>
```

Comment:
The link ends in a 404. The search lists it already, so I guess it'll be reachable soon.

Reply:
Link active now